src/dataflow/src/server/metrics.rs

```diff
@@ -1,4 +1,3 @@
-// Copyright Materialize, Inc. and contributors. All rights reserved.
```

Comment:
This looks problematic!

Reply:
Merge conflict resolution fail. Updated.